### DISCUSSION TITLE
When HitCircle/Slider is selected from Toolbox in Editor, Right-clicking in an empty space should apply New Combo to the next placed object

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#24214] [Ideas] [answered]
Just to make it consistent with Stable

**Comment by peppy** [comment]
Doing WHAT where now? This sounds like a bug. Can you show what you mean in a video?

**Reply by Skubers** [reply]
Here's how it looks in Stable now and has been since I can remember (2014?)

https://github.com/ppy/osu/assets/82095491/44ca0248-f7c1-449a-b046-4014b0ebe864

Notice that as I'm right-clicking, it makes a New Combo for the next object, whereas in Lazer (below) it doesn't

https://github.com/ppy/osu/assets/82095491/9a2ac578-3f10-4610-b0bd-81f7041d97f9

**Reply by peppy** [reply, marked as answer]
I don't think this is coming back, sorry...
